src/test/java/uk/gov/hmcts/reform/wataskmonitor/services/jobs/cleanup/MaintenanceCamundaTaskCleanUpJobServiceTest.java

```diff
@@ -571,4 +583,15 @@ private String getExpectedQueryParametersForActive(String startedBefore) {
 
     }
 
+    private static void assertActualTaskListNotnull(List<HistoricCamundaTask> actualTaskList) {
+        await().atMost(10, TimeUnit.SECONDS)
+            .untilAsserted(() -> assertThat(actualTaskList)
+                .isNotNull());
+    }
+
+    private static void assertActualReportNotNull(GenericJobReport actualReport) {
+        await().atMost(10, TimeUnit.SECONDS)
+            .untilAsserted(() -> assertThat(actualReport)
+                .isNotNull());
```

Comment:
These two methods above are fine as they are.  More for reference or a future improvement we can put these in their own class so they can be re-used and perhaps pass in a function to be used in the `untilAsserted`.  May require some additional refactoring so I'd leave it for now.